apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
Well, I don't think that is possible. This is a UI test which verifies only UI behaviors. In addition, Test is a separate APK running on a test device so I don't think it can check `MobileCenter.isEnable` unless we visualize (toast or whatever) the result on a device. Correct me if I'm wrong.